Dockerfile

```diff
@@ -115,19 +117,23 @@ RUN git config --global http.sslVerify false && \
         -DUSE_ARCADIA_PYTHON=no \
         -DUSE_SYSTEM_PYTHON=3.7\
         -DPYTHON_CONFIG=python3-config \
-        -DCUDA_ROOT=$(dirname $(dirname $(which nvcc)))
-ENV PYTHONPATH=$PYTHONPATH:/opt/catboost/catboost/python-package
+        -DCUDA_ROOT=$(dirname $(dirname $(which nvcc))); \
+        fi
+ENV if ["$CUDA_VERSION" < "11.0"]; then PYTHONPATH=$PYTHONPATH:/opt/catboost/catboost/python-package; fi\
+
+
 
 # xgboost
 RUN git config --global http.sslVerify false && \
     git clone --recursive https://github.com/dmlc/xgboost /opt/xgboost && \
     cd /opt/xgboost && \
     mkdir build && \
     cd build && \
-    cmake .. \
-        -DGPU_COMPUTE_VER="35;50;52;60;61;70" \
+    RMM_ROOT=/opt/conda cmake .. \
+        -DGPU_COMPUTE_VER="60;70;80" \
```

Comment:
I think this is unnecessary: `-DGPU_COMPUTE_VER="60;70;80"` 
xgb auto selects compute capability
https://github.com/dmlc/xgboost/blob/c991eb612d6e4b7936854ac3e142d87fe16ecf72/cmake/Utils.cmake#L92